src/main/java/controllers/MapController.java

```diff
@@ -0,0 +1,86 @@
+package controllers;
+
+import com.google.gson.Gson;
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import database.DatabaseConnection;
+import database.entity.Event;
+import helpers.DateHelper;
+import model.ApiException;
+import model.EventsMapRequest;
+import model.TokenRequest;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.w3c.dom.ranges.Range;
+import org.w3c.dom.ranges.RangeException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.gte;
+
+@Path("/map")
+public class MapController {
+
+    private MongoDatabase database = DatabaseConnection.shared.getDatabase();
+    private static Gson gson = new Gson();
+
+    @Path("/radius")
```

Comment:
Ogólnie w reście ścieżka powinna odzwierciedlać to, co chcesz uzyskać. Z tego endpointu uzyskuje się eventy dla danej pozycji, czyli dla mnie `/events/position?lat=x&lng=y` byłoby opisowe.

Reply:
Podmieniłem na `/events/area?lat=x&lng=y&rad=z`. Wydaje mi się, że opisowo jest git, no i jest też promień obszaru na jakim maja być zwrócone eventy.